src/Components/Blazor/Blazor/src/Services/WebAssemblyJSRuntime.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Mono.WebAssembly.Interop;
+
+namespace Microsoft.AspNetCore.Blazor.Services
+{
+    internal static class WebAssemblyJSRuntime
+    {
+        public static readonly MonoWebAssemblyJSRuntime Current = new MonoWebAssemblyJSRuntime();
```

Comment:
It's a super-minor point, but `Instance` is a more idiomatic name for singletons (which this now is).